src/iuse.cpp

```diff
@@ -3509,10 +3509,15 @@ cata::optional<int> iuse::jackhammer( Character *p, item *it, bool, const tripoi
     }
 
     map &here = get_map();
-    bool mineable_furn = here.has_flag_furn( ter_furn_flag::TFLAG_MINEABLE, pnt );
-    bool mineable_ter = here.has_flag_ter( ter_furn_flag::TFLAG_MINEABLE, pnt );
+    const bool mineable_furn = here.has_flag_furn( ter_furn_flag::TFLAG_MINEABLE, pnt );
+    const bool mineable_ter = here.has_flag_ter( ter_furn_flag::TFLAG_MINEABLE, pnt );
+    const int max_mining_ability = 70;
     if( !mineable_furn && !mineable_ter ) {
         p->add_msg_if_player( m_info, _( "You can't drill there." ) );
+        if( here.bash_resistance( pnt ) > max_mining_ability ) {
+            p->add_msg_if_player( m_info,
+                                  _( "The material is so hard that you hardly can make even a dent." ) );
```

Comment:
Personally I find this sentence a bit confusing. It might be the double hard/hardly appearance of the "hardly can make even". Would something like "This seems so hard you can barely make a dent" be clearer?

Reply:
Or maybe something that strongly implies you'll never mine it like "This material is so hard you can't even make a dent"